tox.ini

```diff
@@ -27,6 +29,9 @@ commands =
   python --version
   coverage run test/runtests.py -v --ignore-files='(^\.|^_|^setup\.py$)' \
                                    --ignore-files=not_imported.py \
+                                   pypy: --ignore-files=_mysqldb_test.py \
+                                   pypy: --ignore-files=db_task_history_test.py \
+                                   pypy: --ignore-files=sqla_test.py \
```

Comment:
instead of using `--ignore-files` is it possible to use `nose.plugins.attr` like we do for the minicluster?